cmd/tls_integration_test.go

```diff
@@ -0,0 +1,67 @@
+//go:build !windows
+// +build !windows
+
+package cmd
+
+// TODO: convert this into the integration tests, once https://github.com/grafana/k6/issues/2459 will be done
+
+import (
+	"encoding/json"
+	"path/filepath"
+	"testing"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type metric struct {
+	Avg float64 `json:"avg"`
+	Min float64 `json:"min"`
+	Max float64 `json:"max"`
+}
+
+type summaryMetrics struct {
+	HTTPReqTLSHandshaking metric `json:"http_req_tls_handshaking"`
+}
+
+type summaryMetricsWrapper struct {
+	Metrics summaryMetrics `json:"metrics"`
+}
+
+func TestTLSLoadSystemCertificates(t *testing.T) {
+	t.Parallel()
+
+	script := `
+		import http from "k6/http";
+
+		export default function() {
+			let response = http.get("https://test-api.k6.io");
```

Comment:
I'm wondering if it's possible to avoid the network/internet call here :thinking:

Reply:
Do you mean creating an https server locally to run the test against ?

Reply:
Yeah, for most of our tests, we use a local HTTP Server. It prevents us from the fails when the network fails. 

But I guess it's not the case with this test, so nevermind :relaxed:

Reply:
We can do something inspired by what we have here https://github.com/grafana/k6/blob/8a74171c8b43f48fb53078e0bcfa7851828124bc/cmd/integration_test.go#L268-L284

@tbourrely feel free to resolve it or we can do it internally after we merged this PR. I unresolve it so we can remember to do it after we have merged if you haven't done it before.

Reply:
done 😉